mkt/games/views.py

```diff
@@ -0,0 +1,23 @@
+from mkt.fireplace.serializers import (FireplaceESAppSerializer,
+                                       FireplaceESWebsiteSerializer)
+from mkt.games.filters import DailyGamesFilter
+from mkt.games.paginator import ESGameAggregationPaginator
+from mkt.search.serializers import DynamicSearchSerializer
+from mkt.search.views import MultiSearchView
+
+
+class DailyGamesView(MultiSearchView):
+    authentication_classes = []
+    cors_allowed_methods = ['get']
+    filter_backends = [DailyGamesFilter]
+    permission_classes = []
+    paginator_class = ESGameAggregationPaginator
+    serializer_class = DynamicSearchSerializer
+
+    def get_serializer_context(self):
```

Comment:
You could inherit from `mkt.fireplace.views.MultiSearchView` instead of having to redefine this.